frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -34,7 +35,11 @@ export const ReportStreamHeader = () => {
     const { authState } = useOktaAuth();
     const { memberships } = useSessionContext();
     const [expanded, setExpanded] = useState(false);
-    let itemsMenu = [<GettingStartedDropdown />, <HowItWorksDropdown />];
+    let itemsMenu = [
+        <GettingStartedDropdown />,
+        <HowItWorksDropdown />,
+        <SupportDropdown />,
```

Comment:
If we're not shipping with content, let's add this dynamically with `CheckFeatureFlag("support")` like we do down on line 42 with Built For You